make.js

```diff
@@ -338,8 +339,8 @@ target.testLegacy = function() {
             // copy the task layout
             var taskJsonPath = path.join(__dirname, 'Tasks', taskName, 'task.json');
             var taskJson = JSON.parse(fs.readFileSync(taskJsonPath).toString());
-            var taskCopySource = path.join(buildPath, taskJson.name);
-            var taskCopyDest = path.join(legacyTestTasksPath, taskJson.name);
+            var taskCopySource = path.join(buildPath, taskName);
```

Comment:
Use folder name instead of task name from task.json.

Reply:
it looks like `taskJsonPath` and `taskJson` arent used anymore

Reply:
Fixed.